package/package.json

```diff
@@ -9,16 +9,16 @@
     "./modal": {
       "import": "./modal.js",
       "require": "./modal.cjs"
+    },
+    "./widget.css": {
+      "import": "./widget.css",
+      "require": "./widget.css"
     }
   },
   "typesVersions": {
     "*": {
-      "modal": [
-        "./modal.d.ts"
-      ],
-      "inline": [
-        "./inline.d.ts"
-      ]
+      "modal": ["./modal.d.ts"],
+      "inline": ["./inline.d.ts"]
```

Comment:
Doesn't have to be included in this PR at all, but can we narrow the the auto-formatting for files only in the `/src` directory. There are many times where these files, often managed by other tools, keep getting flipped back and forth between slightly different formats.